x/staking/keeper/alias_functions.go

```diff
@@ -72,27 +72,22 @@ func (k Keeper) IterateBondedValidatorsByPower(ctx context.Context, fn func(inde
 
 // IterateLastValidators iterates through the active validator set and perform the provided function
 func (k Keeper) IterateLastValidators(ctx context.Context, fn func(index int64, validator types.ValidatorI) (stop bool)) error {
```

Comment:
Would it be possible to discard this function entirely and just use `k.LastValidatorPower.Walk`?